### ISSUE TITLE
lint-action doesn't find golint

### ISSUE THREAD

**Issue by scrouthtv** [wearerequired/lint-action#146]
Hello,

I am getting the same error as in #45 with
```
    - name: Lint Action
      uses: wearerequired/lint-action@v1.7.0
      with:
        github_token: ${{ secrets.GITHUB_TOKEN }}
        golint: true
```

I did install golint before. You can look at the configuration in my repo https://github.com/scrouthtv/card-games/blob/main/.github/workflows/gostyle.yml
First, I install both Go and golint, which also succeeds (https://github.com/scrouthtv/card-games/runs/1680950012)
However, `lint-action` fails to find `golint`.

I think this is because go get installs packages into `$HOME/go/bin` which is not in the path by default.
Did anyone else report this issue? Is there an easy way to fix it?

Thanks for any help in advance

**Comment by ocean90** [comment]
> I think this is because go get installs packages into `$HOME/go/bin` which is not in the path by default.

To test this you could add the path manually to the global with this:

```yml
    - name: Install Go Linter
      run: |
        go get -u golang.org/x/lint/golint
        echo "${HOME}/go/bin" >> $GITHUB_PATH
```

**Comment by scrouthtv** [comment]
Yes, that works.
Thanks a lot.
